scenario/state.py

```diff
@@ -140,6 +140,69 @@ def copy(self) -> "Self":
         return copy.deepcopy(self)
 
 
+class CloudAuthType(str, Enum):
```

Comment:
Maybe link to https://github.com/juju/juju/blob/1c4005149780c7d359c9b3a38c4b0371f9eeb1ed/cloud/clouds.go#L43 in a comment.